nbconvert/exporters/base.py

```diff
@@ -5,7 +5,10 @@
 
 import os
 
-import entrypoints
+try:
```

Comment:
one last thing: because the stdlib version was broken, this is most accurate as a check of `sys.version`, e.g.:

```py
if sys.version_info < (3, 10):
    from importlib_metadata import entry_points
else:
    from importlib.metadata import entry_points
```

https://github.com/jupyterlab/jupyterlab_server/blob/main/jupyterlab_server/translation_utils.py#L20